hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/AbstractRecordStore.java

```diff
@@ -56,6 +57,7 @@
     protected final MapContainer mapContainer;
     protected final RecordFactory recordFactory;
     protected final InMemoryFormat inMemoryFormat;
+    protected final EvictionPolicy evictionPolicy;
```

Comment:
This can be updated in a running cluster, so better to keep it dynamically changeable.

Reply:
I decided to store `EvictionConfig` and then query `evictionConfig.getEvictionPolicy()` for possible runtime updates.